drivers/virtio/ring/include/virtio/virtio_ring.h

```diff
@@ -0,0 +1,196 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
Whee is this file (`virtio_ring.h`) coming from. It seems like a new file. Shouldn't it appear as moved from some place to this place? As is the case with `virtio_ring.c`?